base/settings/R/get_args.R

```diff
@@ -0,0 +1,39 @@
+#' Get Args
+#'
+#' Used in web/workflow.R to parse command line arguments.
+#' See also https://github.com/PecanProject/pecan/pull/2626.
+#' 
+#' @return
+#' @export
+#'
+#' @examples
+#' \dontrun{./web/workflow.R -h}
+get_args <- function () {
+    option_list = list(
+      optparse::make_option(
+        c("-s", "--settings"),
+        default = ifelse(Sys.getenv("PECAN_SETTINGS") != "", 
+                         Sys.getenv("PECAN_SETTINGS"), "pecan.xml"),
```

Comment:
Sys.getenv can itself take a default value, so this simplifies to 

```suggestion
        default = Sys.getenv("PECAN_SETTINGS", "pecan.xml"),
```

But this does miss one edge case: Sys.getenv only uses its default when the variable is unset; in the (perverse?) case where the var is set but its value is the empty string, it will return the empty string. So if it's important that PECAN_SETTINGS="" should result in using pecan.xml, reject this suggestion and keep the existing ifelse.